v63004/u_inref/gtm8202.csh

```diff
@@ -40,7 +38,8 @@ echo "# Expected to return sequences 1"
 $MUPIP journal -forward -extract=./Reg.mjf -seqno=~0,1,~2 "a.mjl" >>& db_log.txt
 echo "# Search extract file for set variables"
 if (  -f Reg.mjf ) then
-	$grep "=" Reg.mjf | $tst_awk -F '\\' '{ print $6 " " $11 }'
+	#$grep "=" Reg.mjf | $tst_awk -F '\\' '{ print $6 " " $11 }'
```

Comment:
Why is this line commented? Can it be removed. Same comments applies to various places in this file.